__tests__/setup-beam.test.js

```diff
@@ -192,6 +196,149 @@ async function testRebar3Versions() {
   assert.deepStrictEqual(got, expected)
 }
 
+async function testGetVersionFromSpec() {
```

Comment:
Definitely not something to do here, but an interesting idea might be to property test functions like these. There are indeed js libs for this. Might be overkill, but makes sense to me 🤷